frontend/src/scenes/ingestion/v2/panels/TeamInvitedPanel.tsx

```diff
@@ -1,31 +1,26 @@
-import { useActions, useValues } from 'kea'
+import { useActions } from 'kea'
 import { ingestionLogicV2 } from 'scenes/ingestion/v2/ingestionLogicV2'
 import { LemonButton } from 'lib/components/LemonButton'
 import './Panels.scss'
 import { LemonDivider } from 'lib/components/LemonDivider'
 import { IconChevronRight } from 'lib/components/icons'
 import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
 import { DemoProjectButton } from './PanelComponents'
-import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
-import { FEATURE_FLAGS } from 'lib/constants'
 
 export function TeamInvitedPanel(): JSX.Element {
     const { completeOnboarding } = useActions(ingestionLogicV2)
     const { reportIngestionContinueWithoutVerifying } = useActions(eventUsageLogic)
-    const { featureFlags } = useValues(featureFlagLogic)
 
     return (
         <div>
             <h1 className="ingestion-title">Help is on the way!</h1>
             <p className="prompt-text">You can still explore PostHog while you wait for your team members to join.</p>
             <LemonDivider thick dashed className="my-6" />
             <div className="flex flex-col mb-6">
-                {featureFlags[FEATURE_FLAGS.ONBOARDING_DEMO_EXPERIMENT] === 'test' ? (
-                    <DemoProjectButton
-                        text="Quickly try PostHog with some demo data."
-                        subtext="Explore insights, create dashboards, try out cohorts, and more."
-                    />
-                ) : null}
+                <DemoProjectButton
```

Comment:
Shouldn't this be behind the new flag you created?

Reply:
I put the check inside that component. https://github.com/PostHog/posthog/pull/13582/files#diff-b45ab9c675aae471e2211c373c0d3bbdb6faaf7f6db73d1a9e72594863f1744eR74